static/app/utils/replays/hooks/useCrumbHandlers.tsx

```diff
@@ -75,7 +82,15 @@ function useCrumbHandlers(startTimestampMs: number = 0) {
   );
 
   const handleClick = useCallback(
-    (crumb: Crumb | NetworkSpan) => {
+    (crumb: Crumb | NetworkSpan | BreadcrumbFrame | SpanFrame) => {
+      if ('offsetMs' in crumb) {
+        const frame = crumb; // Finding `offsetMs` means we have a frame, not a crumb or span
```

Comment:
When/what are crumbs?

Reply:
`Crumb` is the old type name, event & error breadcrumbs (not network data).

Once all the tabs are moved over the types of the args can be narrowed back down